roundabout-core/src/test/java/at.fhv.itm3.s2.roundabout.test/event/CarGenerateEventTest.java

```diff
@@ -0,0 +1,29 @@
+package at.fhv.itm3.s2.roundabout.test.event;
+
+import at.fhv.itm3.s2.roundabout.RoundaboutModel;
+import at.fhv.itm3.s2.roundabout.entity.StreetSection;
+import at.fhv.itm3.s2.roundabout.event.CarGenerateEvent;
+import desmoj.core.simulator.Experiment;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CarGenerateEventTest {
+
+    @Test
+    public void eventRoutineTest() throws Exception {
+        RoundaboutModel model = new RoundaboutModel(null, "", false, false);
```

Comment:
create a new method with the first two lines or so, which creates the model -> reusability